include/boost/url/url.hpp

```diff
@@ -1292,7 +1292,25 @@ class BOOST_SYMBOL_VISIBLE url
     //
     //--------------------------------------------
 
-    /** Normalize everything.
+    /** Normalize the URL components
+
+        Applies Syntax-based normalization to the
```

Comment:
No one knows what "Syntax-based normalization" is. You need to link to the section of the RFC below.